examples/dhf5-custom-hook/src/main/ml-modules/root/custom-modules/ingestion/IngestOrders/archive-hook.sjs

```diff
@@ -6,12 +6,12 @@
 declareUpdate();
 
 // A custom hook receives the following parameters via DHF. Each can be optionally declared.
-var uris; // an array of URIs (may only be one) being processed
-var content; // an array of objects for each document being processed
-var options; // the options object passed to the step by DHF
-var flowName; // the name of the flow being processed
-var stepNumber; // the index of the step within the flow being processed; the first step has a step number of 1
-var step; // the step definition object
+var uris;         // An array of one or more URIs being processed.
+var content;      // An array of objects that represent each document being processed.
+var options;      // The Options object passed to the step by <keyword keyref="MLDH_prodacro"/>.
```

Comment:
I think it can remain 'options' as it refers to the property in json file where it is called 'options'. Also is this <keyword keyref="MLDH_prodacro"/> intended ?

Reply:
The variable names remain the same. Only the comments were changed to match the ones in the documentation.